src/fixed-card-layout.scss

```diff
@@ -0,0 +1,79 @@
+@import './new-settings';
+@import './mixins';
+
+$block: #{$fd-namespace}-fixed-card-group;
+
+.#{$block} {
+  $fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor);
+  $fd-card-margin: 1rem;
+  $fd-card-min-width: 20rem;
+
+  @include fd-reset();
+
+  &--card-layout {
+    @include fd-reset();
+    @include fd-flex();
+
+    justify-content: center;
+
+    * {
+      word-break: break-all;
+    }
+  }
+
+  &--card-layout-column {
+    @include fd-reset();
+
+    flex: 1;
+    margin-right: $fd-card-margin;
+    margin-left: 0;
```

Comment:
do you need this? Isn't it 0 by default?

Reply:
removed